src/components/Match/MatchStory.jsx

```diff
@@ -13,6 +13,16 @@ import styles from './Match.css';
 
 const heroesArr = jsonFn(heroes);
 
+const itemQualityColor = {
+  rare: '#1a87f9',
+  artifact: '#e29b01',
+  secret_shop: '#ffffff',
+  consumable: '#1d80e7',
+  common: '#2bab01',
+  epic: '#b812f9',
+  component: '#ffffff',
+};
```

Comment:
Should this be added in a different place to make it more globally accessible, instead of just in the story tab?
I got these colors from https://www.dota2.com/public/css/heropedia.css